News Theme Development.md

```diff
@@ -129,6 +129,36 @@ Common phrases needed across multiple blocks are declared in global-phrases-bloc
 
 ## Development Process
 
+### Release scripts (ran in github actions)
+
+#### `npm run release:canary` 
+
+- Only publish packages to canary tag
+- Publish packages based on only changes
+- Iterates the prerelease version using `--canary` arg that allows prerelease (not necessarily canary)
+
+
+#### `npm run release:beta` 
+
+- Only publish packages to beta
+- Publish packages based on only changes
+- Iterates the prerelease version using `--canary` arg that allows prerelease (not necessarily canary)
+
+
+#### `npm run release:rc` 
+
+- Only publish packages to rc
+- Publish packages based on only changes
+- Iterates the prerelease version using `--canary` arg that allows prerelease (not necessarily canary)
+
+### `npm run release:stable`
+
+- Only publish packages to stable tag
+- "release:stable": "lerna publish --dist-tag stable -y", better because no force publishes and only changed ones 
+- ideally should go from: (done https://github.com/JackHowa/lerna-monorepo-components/runs/3126215436?check_suite_focus=true)
```

Comment:
Not sure I fully understand this reference

Reply:
yeah that looks like notes; will remove